module-2/backend/week_one.md

```diff
@@ -5,28 +5,92 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+GET to get info from the server
+Post to send info to the server
+DELETE to erease an entry in the server
+PUT replace entry in database

```

Comment:
PUT and PATCH are intended to do basically the same thing - update an existing entry in the database 👍